code/controllers/configuration.dm

```diff
@@ -731,6 +736,7 @@ var/list/storyteller_cache = list()
 				log_misc("Unknown setting in configuration: '[name]'")
 
 /datum/configuration/proc/pick_storyteller(story_name)
+	world << "Configuration calling pick storyteller"
```

Comment:
Debug?

Reply:
yup will fix tomorrow.

Going to bed for now, feel free to leave me a huge pile of requested changes, ill fix em all in a batch later